monai/data/image_reader.py

```diff
@@ -845,8 +846,6 @@ def get_data(
         return patches, metadata
 
     def _check_level(self, img, level):
```

Comment:
I feel maybe it's better to change this function to `_get_level_count()`, then compare `level` and `level_count` at the outside.
What do you think?

Thanks.